README.md

```diff
@@ -22,6 +22,38 @@ Update config.yml and run
 cp2influxdb -c config.yml
 ```
 
+### Systemd
+
+Insert this snippet to the file /lib/systemd/system/cp2influxdb.service:
```

Comment:
This is the wrong path (and we have been wrong for some time) - it must be `/etc/systemd/system/cp2influxdb.service` - the above is for the APT-installed services...